app/javascript/vite/application.js

```diff
@@ -94,3 +94,7 @@ window.SCP.API = ScpApi
 
 window.Spinner = Spinner
 window.morpheus = morpheus
+
+window.AppcuesSettings = {
+  enableURLDetection: true
+}
```

Comment:
Can either this or the same code in `layouts/applicational.html.erb` be removed?